File: Problem Sets/SinaNozhatie-mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{

    int n;

    int j = 1;

    // Picking Posetive number between 1 and 8
    do
    {
        n = get_int("posetive number:");
    }
    while (n > 8 || n < 1);

    for (int i = 1; i <= n; i++) // Height
    {
        int m = n - j; // number of needed space

        for (int k = m; k > 0; k--) // making space

        {
            printf(" ");
        }
        for (j = 1; j <= i; j++) // hash
        {
            printf("#");
        }

        printf("\n");
    }
}

File: Problem Sets/SinaNozhatie-Python-mario-more/mario.py
from cs50 import get_int

# get the Height
while True:
    height = get_int("Height:")
    if height <= 8 and height >= 1:
        break

# Making a loop for the height
for i in range(height):
    # calculating the space     # height - 1 is because the start of i is 0
    space = (height - 1) - i

    # printing  the spaces
    for j in range(space):
        print(" ", end="")

    for n in range(i + 1):
        print("#", end="")

    # printing the 2 space
    print("  ", end="")

    for n in range(i + 1):
        print("#", end="")

    print("")
